docs/guide/doc-index.asciidoc

```diff
@@ -0,0 +1,122 @@
+== Doc index
```

Comment:
I think this page is too detailed, it is going to be hard to maintain, specially for the agents dev. ie. when an agent dev adds or changes something it will have to come here and update it.

I think that the general index page should be enough.

Happy to hear your thoughts.